packages/tpetra/core/src/Tpetra_Import_Util2.hpp

```diff
@@ -164,13 +178,17 @@ sortCrsEntries (const Teuchos::ArrayView<size_t> &CRS_rowptr,
   size_t NumRows = CRS_rowptr.size()-1;
   size_t nnz = CRS_colind.size();
 
+  const bool permute_values_array = CRS_vals.size() > 0;
```

Comment:
This could emit a signed / unsigned comparison warning. Do be sure to test with `-Wall` first.